src/utils/AccessibilityLandmarks.ts

```diff
@@ -0,0 +1,77 @@
+import { $$ } from './Dom';
```

Comment:
My main question while adding the landmarks was my choice of css classes to append the landmarks to. Please focus on these values in your review. Comments/suggestions welcome.

Reply:
Since the implementation of the `tryToSet....Landmark` functions are pretty much the same, maybe you could extract it to a more generic function that you would call multiple times with a cssClass, role & ariaLabel, all defined in an object at the top of the file.